packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2408,6 +2434,17 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
   ///    setting [isSemanticBoundary] to true.
   bool get isSemanticBoundary => false;
 
+  /// Whether this [RenderObject] makes other [RenderObject]s previously painted
+  /// in the same semantics container unreachable for accessibility purposes.
```

Comment:
semantics container should reference isSemanticsBoundary.
...and probably be phrased in terms of semantics boundaries rather than containers.